docker-compose.local.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.7"
+
+services:
+  api:
+    container_name: scrumble-api
+    image: node:12
+    restart: unless-stopped
+    volumes:
+      - .:/server
+    command: sh -c 'cd /server && yarn && yarn start:debug'
+    env_file:
+      - .env
+    environment:
+      POSTGRES_URL: postgresql://scrumble:scrumble@db:5432/scrumble
+    ports:
+      - "3000:3000"
+      - "9229:9229"
+  
+  db:
+    container_name: scrumble-db
+    image: postgres
+    restart: unless-stopped
+    environment:
+      POSTGRES_USER: scrumble
+      POSTGRES_PASSWORD: scrumble
+    ports:
+      - "5432:5432"
```

Comment:
You might want to mount some `db` with volume so that every time you start data will remain